library/ziti_enroll.c

```diff
@@ -149,30 +149,28 @@ int ziti_enroll(ziti_enroll_opts *opts, uv_loop_t *loop, ziti_enroll_cb enroll_c
 static void well_known_certs_cb(char *base64_encoded_pkcs7, ziti_error *err, void *req) {
     ZITI_LOG(DEBUG, "base64_encoded_pkcs7 is: %s", base64_encoded_pkcs7);
 
+    int ziti_err;
     struct ziti_enroll_req *enroll_req = req;
     if ((NULL == base64_encoded_pkcs7) || (NULL != err)) {
         ZITI_LOG(DEBUG, "err->message is: %s", err->message);
         if (enroll_req->enroll_cb) {
-            enroll_req->enroll_cb(NULL, 0, err->code, enroll_req->external_enroll_ctx);
+            enroll_req->enroll_cb(NULL, ZITI_JWT_VERIFICATION_FAILED, err->code, enroll_req->external_enroll_ctx);
```

Comment:
The NULL ziti_context is a bit of a downer, but fine.  Also noticed line 228 in ziti.h describes `ziti_config` parameter of `ziti_enroll_cb` as "identity json data buffer"

Reply:
didn't feel right to pass object back if enrollment failed. Updated doc